brave-jaxrs2/pom.xml

```diff
@@ -37,6 +37,11 @@
             <version>1</version>
         </dependency>
         <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.2</version>
```

Comment:
I don't think it's related.  I had to add this dependency to get access to the `@Priority` annotation as it wasn't available in any of the other dependencies.